app/Filament/Resources/ResultResource.php

```diff
@@ -196,9 +206,17 @@ public static function table(Table $table): Table
                 Tables\Columns\TextColumn::make('download')
                     ->getStateUsing(fn (Result $record): ?string => ! blank($record->download) ? Number::toBitRate(bits: $record->download_bits, precision: 2) : null)
                     ->sortable(),
+                Tables\Columns\TextColumn::make('downloaded_bytes')
+                    ->toggleable()
+                    ->getStateUsing(fn (Result $record): ?string => ! blank($record->download) ? Number::bytesToHuman(bytes: $record->downloaded_bytes, precision: 2) : null)
+                    ->sortable(),
                 Tables\Columns\TextColumn::make('upload')
                     ->getStateUsing(fn (Result $record): ?string => ! blank($record->upload) ? Number::toBitRate(bits: $record->upload_bits, precision: 2) : null)
                     ->sortable(),
+                Tables\Columns\TextColumn::make('uploaded_bytes')
```

Comment:
You can use the `Number::fileSize()` helper provided by Laravel https://laravel.com/docs/12.x/helpers#method-number-file-size here instead of the custom one.